### ISSUE TITLE
[$2000] Empty state icons for `reimbursementAccountLoadingAnimation`  don't have a transparent background in Dark Theme.

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#13151]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Go to any workspace and attempt the Connect Bank Account step
2. Enter the bank account details, go to the company step
3. Fill in the company step, and wait for the reimbursement account loading animation (screen with the title One moment)
4. Check that the icons don't have a transparent background (instead have a gray background).

## Expected Result: 
The Icons should have a transparent background

## Actual Result: 
the icons have a gray background.

## Workaround:
unknown
## Platform:

Where is this issue occurring?

- Mobile Web

**Version Number:** 1.2.33-2
**Reproducible in staging?:** y
**Reproducible in production?:** Could not verify in production
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/204631900-aed363d7-54c5-4b48-8e57-4945501b2378.MP4

https://user-images.githubusercontent.com/43996225/204631989-c4adbb9a-1450-4c50-aa9a-7ffce6c4f6db.MP4

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @mananjadhav 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1669717660117089

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01d555366565521d92</li>
        <li>Upwork Job ID: 1597976370465353728</li>
        <li>Last Price Increase: 2022-12-20</li>
    </ul>
</details>

**Comment by mananjadhav** [comment]
Tagging @Expensify/design  and @shawnborton too

**Comment by abekkala** [comment]
@shawnborton is this going to be fixed internally? 
If so, then I can just create an Upwork posting to hire/pay the contributor who reported the issue.

**Comment by shawnborton** [comment]
The new .gif can be created internally, but we can use a contributor to update the .gif (it's just a file path that points to cloudfront) in the app.

**Comment by Puneet-here** [comment]
@shawnborton, I can make the change (it needs to be done [here](https://github.com/Expensify/App/blob/435c4b93e9e46f3d45b3913a1e30d8c2f2c107bb/src/components/ReimbursementAccountLoadingIndicator.js#L31)). We also need to update other gif that we are using locally  (check the one below)

https://github.com/Expensify/App/blob/main/assets/images/confetti-pop.gif

**Comment by shawnborton** [comment]
Where are we using confetti pop? I thought we stopped using it with the new workspace illustrations that we merged?

**Comment by Puneet-here** [comment]
It's at password screen confirmation and request call confirmation
Once you change your password successfully you will see it or after requesting a call.

**Comment by shawnborton** [comment]
Can you share screenshots of those two pages?

**Comment by Puneet-here** [comment]
<img width="452" alt="Screenshot 2022-11-30 at 5 54 58 AM" src="https://user-images.githubusercontent.com/101883770/204677759-a63fa97b-2cb6-449b-8225-427488217be8.png">
<img width="452" alt="Screenshot 2022-11-30 at 5 55 41 AM" src="https://user-images.githubusercontent.com/101883770/204677856-5f825334-8262-4392-a1fd-4354dba66d10.png">

**Comment by Puneet-here** [comment]
We can use `Illustrations.TadaBlue` instead
```js
            <Icon
                src={Illustrations.TadaBlue}
                height={100}
                width={100}
                fill={defaultTheme.iconSuccessFill}
            />
```
<img width="452" alt="Screenshot 2022-11-30 at 5 58 17 AM" src="https://user-images.githubusercontent.com/101883770/204678454-7d5f2ecf-c7c6-43f7-a653-8d12fc784d6a.png">

and maybe some margin below the icon.

**Comment by shawnborton** [comment]
I think we can use the new simple thumbs up illustration for those:

![image](https://user-images.githubusercontent.com/2319350/204678573-eb4f01de-7a48-4995-bceb-540cf0daf5f8.png)

**Comment by Puneet-here** [comment]
Looks good to me.

<img width="452" alt="Screenshot 2022-11-30 at 6 05 19 AM" src="https://user-images.githubusercontent.com/101883770/204678974-4b7afd42-5914-45ff-9c6d-c086a7209932.png">

**Comment by mananjadhav** [comment]
In this are we covering all the illustrations? One is the `Reimbursement Loading Animation`, and second is the `Request a callback`?

**Comment by Puneet-here** [comment]
> One is the Reimbursement Loading Animation, and second is the Request a callback? 

PasswordConfirmationScreen will be changed too
https://github.com/Expensify/App/blob/11072a6f5cc8bcaaa6e9cad5184ce6adae0df331/src/pages/settings/PasswordConfirmationScreen.js#L18-L19

**Comment by Puneet-here** [comment]
### Proposal 
https://github.com/Expensify/App/blob/b35dba944cff88d5a2d7221dfa626f42d505d1c1/src/pages/RequestCallConfirmationScreen.js#L18-L26
Change the above code with suggested code below -
```diff
-            <Image
-                source={confettiPop}
-                style={styles.confettiIcon}
+            <Icon
+                src={Illustrations.ThumbsUpStars}
+                height={100}
+                width={100}
+                fill={defaultTheme.iconSuccessFill}
             />
             <Text
                 style={[
                     styles.textStrong,
                     styles.textLarge,
-                    styles.mb2,
+                    styles.mv2,
                 ]}
             >
```
https://github.com/Expensify/App/blob/11072a6f5cc8bcaaa6e9cad5184ce6adae0df331/src/pages/settings/PasswordConfirmationScreen.js#L18-L26
same here

```diff 
-            <Image
-                source={confettiPop}
-                style={styles.confettiIcon}
+            <Icon
+                src={Illustrations.ThumbsUpStars}
+                height={100}
+                width={100}
+                fill={defaultTheme.iconSuccessFill}
             />
             <Text
                 style={[
                     styles.textStrong,
                     styles.textLarge,
-                    styles.mb2,
+                    styles.mv2,
                 ]}
```
I think we can create a confirmationPage and we can pass props to it. This way we can reuse it.

**Comment by Puneet-here** [comment]
@shawnborton at [AddPersonalBankAccountPage](https://github.com/Expensify/App/blob/main/src/pages/AddPersonalBankAccountPage.js)
[TransferBalancePage](https://github.com/Expensify/App/blob/main/src/pages/settings/Payments/TransferBalancePage.js) we are using `TadaBlue` should we also change the illustration for them?
<img width="452" alt="Screenshot 2022-11-30 at 2 17 12 PM" src="https://user-images.githubusercontent.com/101883770/204749729-921b4116-f917-4b11-91e8-0334404cee40.png">
<img width="452" alt="Screenshot 2022-11-30 at 2 16 03 PM" src="https://user-images.githubusercontent.com/101883770/204749760-ba5c568a-9057-4e52-afc1-31b1a3a384d3.png">

**Comment by tgolen** [comment]
Can someone please help catch me up on what the current proposal is? I see [this last comment ](https://github.com/Expensify/App/issues/13151#issuecomment-1331821166) from @Puneet-here but it is an incomplete proposal and still leaves a lot of guess-work to be done regarding what is going to change.

**Comment by shawnborton** [comment]
I also wanted to clarify that we (Expensify design team) needs to create these assets internally first before we can assign out to be implemented.

**Comment by shawnborton** [comment]
But yeah, it would be helpful if we could get a concrete list of every place where we need to update one of these empty state images.

**Comment by Puneet-here** [comment]
> it would be helpful if we could get a concrete list of every place where we need to update one of these empty state images.

I have looked into it and we don't have any other place where we have the same issue.

**Comment by shawnborton** [comment]
Yeah, I think we should update the original issue comment with all of the places that need to be updated though so that this issue is more clear.

**Comment by abekkala** [comment]
> Yeah, I think we should update the original issue comment with all of the places that need to be updated though so that this issue is more clear.

@shawnborton is that something that you're looking for a contributor to do? or is this internal until after that point for fixing?

**Comment by abekkala** [comment]
@shawnborton ok, should this just be placed on [HOLD] until then as there is no immediate progress that can be made by the contributor at this time?

**Comment by shawnborton** [comment]
Oh yeah, good call. Let's put a hold on it and I will update once we have new assets.

**Comment by abekkala** [comment]
I'll move back to Daily once this is off hold

**Comment by tgolen** [comment]
@shawnborton do you happen to have an ETA for those assets? I'm sure you've been busy, but just trying to have the right expectations for this one.

**Comment by shawnborton** [comment]
I think we might be getting them this Friday, if not, then by next Friday is my hope!

**Comment by shawnborton** [comment]
Okay we have files!

For the account reviewing animation, we show it at 140px by 140px, and here is the .gif:
![Expensify-Reviewing-GREEN-121322-kjs-1_1](https://user-images.githubusercontent.com/2319350/208327173-553ae140-37d9-4270-9ad2-5fcee07af10a.gif)

For the celebration image, let's use this new gif and show it at 180px by 180px:
![Expensify-Fireworks-GREEN-121322-kjs-1](https://user-images.githubusercontent.com/2319350/208327192-9b22fd8e-17de-442d-a793-cb805ec995f0.gif)

Here are the files: [AnimatedGIFs.zip](https://github.com/Expensify/App/files/10255012/AnimatedGIFs.zip)


Note that we'll want to follow up and replace these with Lottie animations when the time comes (cc @roryabraham @JmillsExpensify) - and we have the Lottie JSON exports ready to go already.

**Comment by abekkala** [comment]
@shawnborton is this good to be taken off hold for the Upwork job? 
(https://www.upwork.com/jobs/~01d555366565521d92)

I'm now ooo and will need to reassign this. For the new assignee: 
Upwork job is already created.
We may need to take the title off hold as the files have come in. 
@tgolen and/or @tgolen will be hiring their chosen contributor

**Comment by shawnborton** [comment]
Yup, I think this is ready to be implemented so feel free to take it off hold!

**Comment by joekaufmanexpensify** [comment]
Sounds good! Took off hold. I'll look out for confirmation of the contributor handling this so they can be hired in upwork.

**Comment by shawnborton** [comment]
The videos look great! I don't feel strongly as to whether or not we should keep the images hosted where they are on Cloudfront or bundle the up with the app. Thoughts @tgolen ?

**Comment by Puneet-here** [comment]
### Proposal

As [proposed earlier](https://github.com/Expensify/App/issues/13151#issuecomment-1331772208) we should create a confirmation component because we are using it at multiple places -
[PasswordConfirmationScreen](https://github.com/Expensify/App/blob/main/src/pages/settings/PasswordConfirmationScreen.js)
[RequestCallConfirmationScreen](https://github.com/Expensify/App/blob/main/src/pages/RequestCallConfirmationScreen.js)
[AddPersonalBankAccountPage](https://github.com/Expensify/App/blob/main/src/pages/AddPersonalBankAccountPage.js)
[TransferBalancePage](https://github.com/Expensify/App/blob/main/src/pages/settings/Payments/TransferBalancePage.js)
[ActivateStep](https://github.com/Expensify/App/blob/main/src/pages/EnablePayments/ActivateStep.js)



We just have to import the newly created confirmation page component to all the above pages and we just need to pass the needed props. We can add the gif given above as default prop.

This is how the main code will look like- 
```js
 <>
        <View style={[styles.screenCenteredContainer, styles.alignItemsCenter]}>
            <Image
                source={props.illustration}
                style={styles.confirmationAnimation}
            />
            <Text
                style={[
                    styles.textStrong,
                    styles.textLarge,
                    styles.mv2,
                ]}
            >
                {props.heading}
            </Text>
            <Text style={styles.textAlignCenter}>
                {props.description}
            </Text>
        </View>
        {props.shouldShowButton && (
        <FixedFooter>
            <Button
                success
                text={props.buttonText}
                style={styles.mt6}
                pressOnEnter
                onPress={props.onButtonPress}
            />
        </FixedFooter>
        )}
    </>
);
```
At styles we need to make the suggested changes -
For our new confirmation page component -
```diff
-    confettiIcon: {
-        height: 100,
-        width: 100,
+    confirmationAnimation: {
+        height: 180,
+        width: 180,
         marginBottom: 20,
     },

```
For https://github.com/Expensify/App/blob/main/src/components/ReimbursementAccountLoadingIndicator.js -
```diff
     loadingVBAAnimation: {
-        width: 160,
-        height: 160,
+        width: 140,
+        height: 140,
     },
```
and as [mentioned earlier](https://github.com/Expensify/App/issues/13151#issuecomment-1331473525) we just have to import the newly provided gif and pass it in the `source` prop
https://github.com/Expensify/App/blob/e36a3fafc73006b394c8257b1d0413ea6b90750c/src/components/ReimbursementAccountLoadingIndicator.js#L31
The gifs will be added to the `assets/images`

**Comment by tgolen** [comment]
@Puneet-here I like your proposal of turning this into a component that's used consistently across the app. Let's keep running with that!

In fact, I think the proposal is good enough to move forward with it, so 🟢 

The only reason I would avoid putting the gifs into the application (vs putting them in cloudfront) is to keep the size of the bundled app smaller. I think that since the current gifs are in cloudfront, let's have the new ones in cloudfront too.

Also, is there any benefit in using `<Image>` instead of `<Icon>`? I see both of them used on those pages.

**Comment by thesahindia** [comment]
I also like the idea of making a common component (thought of doing so when created password confirmation page)

**Comment by joekaufmanexpensify** [comment]
Assigning @Puneet-here based on [confirmation](https://github.com/Expensify/App/issues/13151#issuecomment-1359903533) that we can move forward with their proposal.

**Comment by joekaufmanexpensify** [comment]
We need to hire:

- @Puneet-here - $2,000 for the PR. 
- @sobitneupane - $2,000 for C+ PR review. 
- @mananjadhav - $250 for reporting the bug.

**Comment by joekaufmanexpensify** [comment]
@Puneet-here offer sent!

**Comment by joekaufmanexpensify** [comment]
@sobitneupane offer sent!

**Comment by joekaufmanexpensify** [comment]
@mananjadhav offer sent!

**Comment by Puneet-here** [comment]
I have created a [draft](https://github.com/Expensify/App/pull/13744), I haven't added the gif for [ReimbursementAccountLoadingIndicator.js](https://github.com/Expensify/App/blob/main/src/components/ReimbursementAccountLoadingIndicator.js) because we want to use the cloudfront url instead (let me know once we have uploaded the gif) I have added the second gif to the assets for now but it also needs to be uploaded in cloudfront (i will change the code once that's done)

**Comment by Puneet-here** [comment]
The only page that is left is `ActivateStep` it contains two icons. Now that we are using gif, the ConfirmationPage component contains `<Image>` component which won't be able to render the svg so I think we should also upload a gif for `Illustrations.ReceiptsSearchYellow` https://github.com/Expensify/App/blob/9d357e459b1b0114d60f5b6162ea6ee9672356aa/src/pages/EnablePayments/ActivateStep.js#L40
This is how the current icon looks
<img width="365" alt="Screenshot 2022-12-21 at 4 10 43 AM" src="https://user-images.githubusercontent.com/101883770/208781675-570830fd-1028-41e3-9778-ba5d76ba30c1.png">

cc: @shawnborton @tgolen

**Comment by shawnborton** [comment]
I agree, let's update it there as well.

**Comment by Puneet-here** [comment]
We are waiting for gif to replace `Illustrations.ReceiptsSearchYellow`  and we are waiting for the gifs to be uploaded to the cloudfront
@joekaufmanexpensify, can we hold the PR merge timeline because of the above since we need to wait before making any new changes?

**Comment by shawnborton** [comment]
I will help you get the assets uploaded to Cloudfront - will keep you posted when they are available.

**Comment by shawnborton** [comment]
Here you go:
- https://d2k5nsl2zxldvw.cloudfront.net/images/animations/animation_accountreview.gif
- https://d2k5nsl2zxldvw.cloudfront.net/images/animations/animation__fireworks.gif

**Comment by joekaufmanexpensify** [comment]
@Puneet-here is the PR still on hold, or can you now proceed as Shawn linked the updated cloudfront GIF links?

**Comment by Puneet-here** [comment]
> can you now proceed as Shawn linked the updated cloudfront GIF links

Yes, I can proceed. I am making the changes right now.

**Comment by Puneet-here** [comment]
It is ready for the review @sobitneupane.

@joekaufmanexpensify, can we ask another engineer for help? I just found that @tgolen is not available.

**Comment by joekaufmanexpensify** [comment]
@Puneet-here I [asked](https://expensify.slack.com/archives/C03TQ48KC/p1671728822292809) about this internally to see if someone else can step in to help. 

However, given the holidays in the US, it is possible that no one else will be able to step in. A lot of our team has some OOO in the coming days. If that is the case, we'll have a discussion internally about how to take our internal OOO time into account when calculating the time it took for the PR to be merged.

**Comment by joekaufmanexpensify** [comment]
Okay @mountiny confirmed they can assist with reviewing the PR, so assigning them to the issue as well!

**Comment by JmillsExpensify** [comment]
Nice! Also, I'm happy to keep an eye on this one while you're out. Will help move things along, if required.

**Comment by joekaufmanexpensify** [comment]
Sounds great, thank you!

**Comment by mountiny** [comment]
Merged 🎉 nice job @Puneet-here

**Comment by Puneet-here** [comment]
Thanks for the help reviewing it.

**Comment by JmillsExpensify** [comment]
Nice, we'll circle back on this for payment as the regression period passes.

**Comment by joekaufmanexpensify** [comment]
Hm, looks like the bug zero checklist / payment notification was not added to this issue when the PR was deployed to production 5 days ago? I'm thinking maybe they weren't added because I manually changed the issue to weekly? 

Regardless, I will manually create the BZ checklist, and payment checklist on the issue. Payment will be issued on 2023-01-05. 

<img width="860" alt="image" src="https://user-images.githubusercontent.com/54866469/210367017-302db26f-f859-4f22-b985-cb36d52a0745.png">

**Comment by joekaufmanexpensify** [comment]
The solution for this issue has been 🚀 deployed to production 🚀 in version 1.2.44-0 and is now subject to a 7-day regression period 📆. Here is the list of pull requests that resolve this issue:

https://github.com/Expensify/App/pull/13859

If no regressions arise, payment will be issued on 2023-01-05. 🎊

After the hold period, please check if any of the following need payment for this issue, and if so check them off after paying:

- [ ] External issue reporter
- [ ] Contributor that fixed the issue
- [ ] Contributor+ that helped on the issue and/or PR

As a reminder, here are the bonuses/penalties that should be applied for any External issue:
- Merged PR within 3 business days of assignment - 50% bonus
- Merged PR more than 9 business days after assignment - 50% penalty

**Comment by joekaufmanexpensify** [comment]
Okay 7 day regression period has passed, and we're all set to issue payment here. Given the PR was merged within 3 business days from when the contributor was assigned to the issue, both the contributor and C+ qualify for a 50% bonus here. We need to pay the following amounts:

- @mananjadhav - $250 for reporting bug.
- @Puneet-here - $3,000 for PR fixing the bug (includes 50% bonus of $1,000). 
- @sobitneupane - $3,000 for C+ review (includes 50% bonus of $1,000).

**Comment by joekaufmanexpensify** [comment]
@mananjadhav $250 sent, and contract ended!

**Comment by joekaufmanexpensify** [comment]
@Puneet-here $3,000 sent, and contract ended!

**Comment by joekaufmanexpensify** [comment]
@sobitneupane $3,000 sent, and contract ended!

**Comment by joekaufmanexpensify** [comment]
Job closed in upwork.

**Comment by joekaufmanexpensify** [comment]
Proposed new regression test [here](https://expensify.slack.com/archives/C01SKUP7QR0/p1672951545887609)!

**Comment by mountiny** [comment]
To complete the checklist, I have commented on this Pr where we have made the switch to the dark mode https://github.com/Expensify/App/pull/12944#issuecomment-1373329230 and we should have probably update the icons there or at least have a follow up ready.

Posted a discussion in Slack https://expensify.slack.com/archives/C049HHMV9SM/p1672994272661819

I think the regression tests Joe posted should be enough for this going forwards.

**Comment by joekaufmanexpensify** [comment]
Added new regression test issue above!

**Comment by joekaufmanexpensify** [comment]
Okay, bug is fixed, payment is handled, and BZ checklist is complete. This is all set. Thanks everyone!
